src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -36,11 +38,25 @@ public class AadIssuerValidator
             IsV2Authority = aadAuthority.Contains(V2EndpointSuffix);
         }
 
+        internal AadIssuerValidator(
+            HttpClient httpClient,
```

Comment:
does it make sense to accept the http client here? It looks like it will be unused?